hedera-node/test-clients/src/main/java/com/hedera/services/bdd/junit/hedera/utils/GrpcUtils.java

```diff
@@ -109,93 +118,115 @@ public static TransactionResponse submit(
             @NonNull final SystemFunctionalityTarget target,
             @NonNull final com.hederahashgraph.api.proto.java.AccountID nodeAccountId) {
         return switch (functionality) {
-            case ContractCall -> clients.getScSvcStub(nodeAccountId, false).contractCallMethod(transaction);
-            case ContractCreate -> clients.getScSvcStub(nodeAccountId, false).createContract(transaction);
-            case ContractUpdate -> clients.getScSvcStub(nodeAccountId, false).updateContract(transaction);
-            case ContractDelete -> clients.getScSvcStub(nodeAccountId, false).deleteContract(transaction);
-            case EthereumTransaction -> clients.getScSvcStub(nodeAccountId, false)
+            case ContractCall -> clients.getScSvcStub(nodeAccountId, false, false)
+                    .contractCallMethod(transaction);
+            case ContractCreate -> clients.getScSvcStub(nodeAccountId, false, false)
+                    .createContract(transaction);
+            case ContractUpdate -> clients.getScSvcStub(nodeAccountId, false, false)
+                    .updateContract(transaction);
+            case ContractDelete -> clients.getScSvcStub(nodeAccountId, false, false)
```

Comment:
For these and all of the following transaction `case` statements, it seems cleaner to add simple method overloads to `HapiClients`:

```java
public FileServiceBlockingStub getFileSvcStub(final AccountID nodeId, final boolean useTls) {
    return getFileSvcStub(nodeId, useTls, false);
}
```

This way we wouldn't have to add another `false` param to a bunch of transaction calls that have nothing to do with node operator queries. Just a thought.

Reply:
Can we do that with a separate task?

Reply:
Sure, whatever's easiest. Making this change now would eliminate this entire block of changes (lines 121 - 229), that's mainly why I suggested it, but doing it later is fine too :)